packages/dashboard/src/components/customOrangeButton/index.tsx

```diff
@@ -8,8 +8,16 @@ import './index.css';
 const CustomOrangeButton = ({
   title,
   handleClick,
+  disabled,
   ...rest
 }: { title: string; handleClick?: () => void } & ButtonProps) => {
+  if (disabled) {
```

Comment:
at this point should we just make our own orange button so we're not overriding all the cloudscape settings for every possible state

Reply:
this is the only state we need explicitly, i think it should be ok for now... any more customization, we probably should fall back to the "blue" primary.. instead the internal orange promary